src/main/java/core/basesyntax/FileWork.java

```
@@ -8,7 +18,33 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String REGEX = "\\b[wW]\\w*\\b";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        Path file = Paths.get(fileName);
+        List<String> allLines = null;
+        try {
+            allLines = Files.readAllLines(file);
+        } catch (IOException e) {
+            e.printStackTrace();
```

don't ignore exceptions